### ISSUE TITLE
Override of old template files to new isn't recognized by Woocommerce

### ISSUE THREAD

**Issue by jccbbb** [woothemes/woocommerce#35911]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

My Woocommerce installation (7.1.1) is giving me the following warning in Woocommerce -> Status.
`<THEMENAME>/woocommerce/cart/cart.php version 3.8.0 is out of date. The core version is 7.0.1`

I have downloaded the latest Woocommerce-7.1.1.zip and extracted out the cart.php and replaced the old on with it.one

### Expected behavior

The warning should disappear since I have updated it to the latest template file.

### Actual behavior

The same warning persists.

### Steps to reproduce

Pre-req: Having a cart.php file from an older version of Woocommerce that needs to be replaced.

1. Go to Woocommerce -> Status
2. Go to Overrides section

### WordPress Environment
6.1.1
`
### wp-core ###

version: 6.1.1
site_language: en_GB
user_language: en_GB
timezone: Europe/London
permalink: /%postname%/
https_status: false
multisite: false
user_registration: 0
blog_public: 0
default_comment_status: open
environment_type: production
user_count: 3
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /var/www/html
wordpress_size: 54.55 MB (57202503 bytes)
uploads_path: /var/www/html/wp-content/uploads
uploads_size: 3.45 MB (3618010 bytes)
themes_path: /var/www/html/wp-content/themes
themes_size: 429.20 MB (450044495 bytes)
plugins_path: /var/www/html/wp-content/plugins
plugins_size: 71.01 MB (74460067 bytes)
database_size: 5.59 MB (5865472 bytes)
total_size: 563.80 MB (591190547 bytes)

### wp-active-theme ###

name: Base theme (<THEMENAME>)
version: 1.0.0
author: <HIDDEN>
author_website: (undefined)
parent_theme: none
theme_features: core-block-patterns, widgets-block-editor, post-thumbnails, automatic-feed-links, html5, editor-color-palette, woocommerce, wc-product-gallery-lightbox, wc-product-gallery-slider, menus, title-tag, custom-logo, wp-block-styles, align-wide, editor-styles, editor-style, responsive-embeds
theme_path: /var/www/html/wp-content/themes/<THEMENAME>
auto_update: Disabled

### wp-plugins-active (8) ###

Advanced Custom Fields PRO: version: 6.0.5, author: WP Engine, Auto-updates disabled
PaysonCheckout for WooCommerce: version: 3.4.4, author: Krokedil, Auto-updates disabled
Product Customer List for WooCommerce Premium: version: 2.9.2, author: Kokomo, Auto-updates disabled
WooCommerce: version: 7.1.1, author: Automattic, Auto-updates disabled
WooCommerce Attribute Stock: version: 1.7.12, author: Mewz (latest version: 1.8.3), Auto-updates disabled
WooCommerce Custom Fields: version: 2.3.4, author: RightPress, Auto-updates disabled
WooCommerce Stripe Gateway: version: 7.0.1, author: WooCommerce, Auto-updates disabled
Woo Swish e-commerce: version: 3.5.6, author: BjornTech, Auto-updates disabled

### wp-plugins-inactive (3) ###

Akismet Anti-Spam: version: 4.1.9, author: Automattic (latest version: 5.0.2), Auto-updates disabled
Password Protected: version: 2.5.3, author: WPExperts, Auto-updates disabled
WooCommerce Attribute Stock (Lite): version: 1.7.12, author: Mewz, Auto-updates disabled

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1690
imagemagick_version: ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org
imagick_version: 3.4.4
file_uploads: File uploads is turned off
post_max_size: 8M
upload_max_filesize: 2M
max_effective_size: 2 MB
max_file_uploads: 20
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 122 MB
	imagick::RESOURCETYPE_DISK: 1073741824
	imagick::RESOURCETYPE_FILE: 786432
	imagick::RESOURCETYPE_MAP: 512 MB
	imagick::RESOURCETYPE_MEMORY: 256 MB
	imagick::RESOURCETYPE_THREAD: 8
imagemagick_file_formats: 3FR, 3G2, 3GP, AAI, AI, ART, ARW, AVI, AVS, BGR, BGRA, BGRO, BIE, BMP, BMP2, BMP3, BRF, CAL, CALS, CANVAS, CAPTION, CIN, CIP, CLIP, CMYK, CMYKA, CR2, CRW, CUR, CUT, DATA, DCM, DCR, DCX, DDS, DFONT, DNG, DPX, DXT1, DXT5, EPDF, EPI, EPS, EPS2, EPS3, EPSF, EPSI, EPT, EPT2, EPT3, ERF, FAX, FILE, FITS, FRACTAL, FTP, FTS, G3, G4, GIF, GIF87, GRADIENT, GRAY, GRAYA, GROUP4, H, HALD, HDR, HEIC, HISTOGRAM, HRZ, HTM, HTML, HTTP, HTTPS, ICB, ICO, ICON, IIQ, INFO, INLINE, IPL, ISOBRL, ISOBRL6, J2C, J2K, JBG, JBIG, JNG, JNX, JP2, JPC, JPE, JPEG, JPG, JPM, JPS, JPT, JSON, K25, KDC, LABEL, M2V, M4V, MAC, MAGICK, MAP, MASK, MAT, MATTE, MEF, MIFF, MKV, MNG, MONO, MOV, MP4, MPC, MPEG, MPG, MRW, MSL, MTV, MVG, NEF, NRW, NULL, ORF, OTB, OTF, PAL, PALM, PAM, PATTERN, PBM, PCD, PCDS, PCL, PCT, PCX, PDB, PDF, PDFA, PEF, PES, PFA, PFB, PFM, PGM, PGX, PICON, PICT, PIX, PJPEG, PLASMA, PNG, PNG00, PNG24, PNG32, PNG48, PNG64, PNG8, PNM, PPM, PREVIEW, PS, PS2, PS3, PSB, PSD, PTIF, PWP, RADIAL-GRADIENT, RAF, RAS, RAW, RGB, RGBA, RGBO, RGF, RLA, RLE, RMF, RW2, SCR, SCT, SFW, SGI, SHTML, SIX, SIXEL, SPARSE-COLOR, SR2, SRF, STEGANO, SUN, TEXT, TGA, THUMBNAIL, TIFF, TIFF64, TILE, TIM, TTC, TTF, TXT, UBRL, UBRL6, UIL, UYVY, VDA, VICAR, VID, VIFF, VIPS, VST, WBMP, WEBP, WMV, WPG, X, X3F, XBM, XC, XCF, XPM, XPS, XV, XWD, YCbCr, YCbCrA, YUV
gd_version: bundled (2.1.0 compatible)
gd_formats: GIF, JPEG, PNG, BMP
ghostscript_version: 9.27

### wp-server ###

server_architecture: Linux 5.15.79.1-microsoft-standard-WSL2 x86_64
httpd_software: Apache/2.4.38 (Debian)
php_version: 7.4.20 64bit
php_sapi: apache2handler
max_input_variables: 1000
time_limit: 30
memory_limit: 128M
admin_memory_limit: 256M
max_input_time: -1
upload_max_filesize: 2M
php_post_max_size: 8M
curl_version: 7.64.0 OpenSSL/1.1.1d
suhosin: false
imagick_availability: true
pretty_permalinks: true
htaccess_extra_rules: false

### wp-database ###

extension: mysqli
server_version: 5.7.34
client_version: mysqlnd 7.4.20
max_allowed_packet: 4194304
max_connections: 151

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /var/www/html/wp-content
WP_PLUGIN_DIR: /var/www/html/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: true
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_ENVIRONMENT_TYPE: Undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

`

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by swatipawarGS** [comment]
Hi @jccbbb,
Thank you for taking the time to report this bug, we really appreciate your help. We are unable to reproduce the issue on our end using the WooCommerce 7.1.1 version and WordPress 6.1.1 version. 

Please find below the screenshot for reference :
![#35911](https://user-images.githubusercontent.com/94531721/206981053-03ed5b18-fc0c-4892-aaff-0e46c5afa21c.jpg)

Please provide us with more details and any additional steps required to repro this issue which may help us evaluate it further.

**Comment by syllie** [comment]
Hi @roykho 

I arrived here having exactly the same issue, and I also had a look on the community forums, which on each thread have multiple 'hey that affects me too' responses:

https://wordpress.org/support/topic/outdated-template-override-warning-that-seems-invalid/
https://wordpress.org/support/topic/woocommerce-template-is-out-of-date-but-it-isnt/
https://wordpress.org/support/topic/template-files-not-overwriting/

My situation is exactly as OP:
 1. I updated WooCommerce to the latest version, checked the status page
 2. My theme got two template out of date warnings, indicating they were out of sync with the core versions.
 3. I updated the templates, including the version numbers
 4. Refresh of the status page: out of date warning persists.
 5. Clear template cache -> does not resolve
 Other caching: disabled.
 
From the OP post, the community posts, as well as my own experience - there seems to be something not working correctly.

I have updated these files many times in the past (my theme is > 10 yrs old) and this is the first time I ran into this issue.
